port/posix/posix_transport_shm.h

```diff
@@ -29,31 +54,57 @@
 #include "wolfhsm/wh_settings.h"
 
 #include <stdint.h>
+#include <limits.h>
 
-#include "wolfhsm/wh_transport_mem.h"
 #include "wolfhsm/wh_comm.h"
+#include "wolfhsm/wh_transport_mem.h"
 
-/** Common configuration structure */
+/** POSIX SHM configuration structure */
 typedef struct {
-    char*    shmObjName; /* Null terminated, up to NAME_MAX */
-    uint16_t req_size;
-    uint16_t resp_size;
-    uint8_t  WH_PAD[4];
+    char*       name; /* Null terminated, up to NAME_MAX */
+    size_t      dma_size;
+    uint16_t    req_size;
+    uint16_t    resp_size;
 } posixTransportShmConfig;
 
 
-/** Common context */
+/** POSIX SHM context */
+typedef enum {
+    PTSHM_STATE_NONE = 0,       /* Not opened, not mapped */
+    PTSHM_STATE_MAPPED,         /* Opened and mapped, no client */
+    PTSHM_STATE_INITIALIZED,    /* Fully initialized */
+    PTSHM_STATE_DONE,           /* Closed and complete */
```

Comment:
This state is never set by client or server

Reply:
Updated.  Now set during cleanup.